Cartfile

```diff
@@ -13,4 +13,4 @@ github "mozilla-mobile/MappaMundi"          "master"
 github "mozilla-services/shavar-prod-lists" "89.0"
 
 # Use release version of application-services
-github "mozilla/application-services"      "v85.0.0"
+github "mozilla/application-services"      "v85.2.0"
```

Comment:
I checked bitrise and it failed:
```
 shell task (/usr/bin/env git checkout --quiet --force v85.2.0 (launched in /Users/vagrant/Library/Caches/org.carthage.CarthageKit/dependencies/application-services)) failed with exit code 1:
error: pathspec 'v85.2.0' did not match any file(s) known to git
```

I think they have used a capital V for the version instead of v... https://github.com/mozilla/application-services/releases